### ISSUE TITLE
Allow list item prefixes to be overwritten on “Do’s & Don’ts” lists

### ISSUE THREAD

**Issue by gazjoy** [nhsuk/nhsuk-frontend#539]
## What is the issue?

Not so much a bug as something that would be helpful. I am using the “Do’s & Don’ts” lists nunjucks macro and cannot find a way of overwriting the "do" and "don't" prefix. The lists with the ticks and crosses would be helpful for me working on the NHSBSA NHS pension service to show users a list of factors that increase or reduce their pension. This is just an idea of use at the moment and I get that the component is called “Do’s & Don’ts” lists for a reason. It works well in the clinical context.

We at NHSBSA are in the process of collating components and patterns for our own needs so maybe this is something that lives in there instead?

## What steps are required to reproduce the issue?

Use the “Do’s & Don’ts” lists macro, the prefixes are hard coded.

## What was the environment where this issue occurred?

* Device: Macbook Pro
* Operating System: MacOS Mojave
* Browser: Chrome
* Browser version: 77.0.3865.90
* NHS.UK frontend package version: 3.0.3
* Node version: 8.11.3
* npm version: 6.5.0

## Is there anything else you think would be useful in recreating the issue?

Aim:
![image](https://user-images.githubusercontent.com/628064/66380544-992e2e80-e9af-11e9-9b8c-82ea27dec6e2.png)

Currently implemented with HTML rather than macro.

**Comment by magicaljellybeans** [comment]
Has this already been solved?

 The above could be implemented with:
```
{{ list({
  "title": "Increase it",
  "type": "tick",
  "items": [
    {
      "item": "Time served"
    },
    {
      "item": "Retire one year later"
    },
    {
      "item": "Reduced lump sum"
    }
  ]
}) }}
```

**Comment by AdamChrimes** [comment]
This will be fixed in release 5.0.0 (next week)
